### PR TITLE
Release v3.4.0

### PR DESCRIPTION
**Pull request by BartoszSiwinski**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Release notes
 
+## New in git-machete 3.4.0
+
+- added: `github` command with `anno-prs`, `checkout-prs`, `create-pr` and `retarget-pr` subcommands
+- added: documentation on readthedocs.io
+- fixed: documentation displayed with `help`/`-h`/`--help`
+- improved: content of git-machete project related blogs has been moved to this repo and updated
+
+## New in git-machete 3.3.1
```
Comment (PawelLipski) [inline RELEASE_NOTES.md]: Whoops there's never been version 3.3.1, pls merge this section into 3.4.0 (separate PR to develop)

File: tox.ini [hunk 2]
```diff
@@ -30,5 +28,32 @@ commands = {posargs}
...
+  sphinx-rtd-theme==0.5.2
+commands =
+  sphinx-build -b html docs/source docs/html
+
+[testenv:mypy]
+whitelist_externals = tox
+commands = tox -e "mypy-py{36,37,38,39}"
+
+[testenv:mypy-py{36,37,38,39}]
+whitelist_externals = mypy
```
Comment (PawelLipski) [inline tox.ini]: Hmmm shouldn't `mypy` be in `deps` here? It looks like now it implicitly relies on `mypy` already being installed (presumably) via a previous execution of a `testenv`... not critical ofc, can be addressed in a subsequent release

Comment (BartoszSiwinski) [reply tox.ini]: There is no reason to delay this fix since I'm already creating PR to fix the release notes. I will include that fix.

**Review by PawelLipski — APPROVED** [review]
:haha-riliz:

**Pull request merged**